modules/libpref/init/all.js

```diff
@@ -807,7 +807,7 @@ pref("toolkit.dump.emit", false);
 // Address of the cloud server to connect to for record/replay sessions.
 pref("devtools.recordreplay.cloudServer", "wss://dispatch.replay.io");
 pref("devtools.recordreplay.recordingsUrl", "https://app.replay.io/");
-pref("devtools.recordreplay.usePreallocated", false);
+pref("devtools.recordreplay.disablePreallocated", false);
```

Comment:
This looks good, but usually a patch like this would just flip the default value for the pref instead of renaming it.

Reply:
I know, that's the typical thing.  But I wanted to forestall any doubt later on about "was it really turned on for me or did I have a sticky setting from before?".